MOxUnit/util/moxunit_util_escape_xml.m

```diff
@@ -0,0 +1,33 @@
+function str_esc=moxunit_util_escape_xml(str)
```

Comment:
I don't think so... I would assume that everything has to be escaped, also strings within double quotes.